internal/cli/opsmanager/clusters/unmanage.go

```diff
@@ -75,7 +75,6 @@ func UnmanageBuilder() *cobra.Command {
 	}
 	cmd := &cobra.Command{
 		Use:     "unmanage <name>",
-		Aliases: []string{"rm"},
```

Comment:
This alias name was also aliased in the "delete" subcommand which was shadowing this one, so removing this alias should not be breaking.